src/main/proto/user-service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package sjchat.users;
+
+option java_multiple_files = true;
+option java_package = "sjchat.users";
+option java_outer_classname = "UserServiceProto";
+
+service UserService {
+    rpc createUser (UserDataRequest) returns (UserResponse);
+    rpc getUser (UserRequest) returns (UserResponse);
+    rpc updateUser (UserDataRequest) returns (UserResponse);
+}
+
+message User {
+    int64 id = 1;
+    string username = 2;
+}
+
+message UserDataRequest {
```

Comment:
The proto definitions are a great place to add documentation about the API. For example, what is the semantics of the updateUser call? Can I change something w.r.t a user by only giving it the username? (this is unclear now).